src/python/WMComponent/JobUpdater/JobUpdaterPoller.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, config):
         self.config = config
 
         self.bossAir = BossAirAPI(config = self.config)
-        self.reqmgr = RequestManager({'endpoint' : self.config.JobUpdater.reqMgrUrl})
+        self.reqmgr = ReqMgr({'endpoint' : self.config.JobUpdater.reqMgrUrl})
```

Comment:
Before instantiating this class, shouldn't we check whether the config has ReqMgr or ReqMgr2?
Basically the same thing that you applied to CleanCouchPoller